app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -164,11 +170,7 @@ class ExposureNotificationsRepository(
                 hmackey,
                 null,
                 prefs.getRevisionToken(),
-                healthAuthorityID = if (BuildConfig.FLAVOR == "dev") {
-                    "cz.covid19cz.erouska.dev"
```

Comment:
Nepřestane pak fungovat DEV verze?

Reply:
nn, to byl workaround pro odesílání bez verifikace a to už neděláme